### ISSUE TITLE
Preset for everything except formatting

### ISSUE THREAD

**Issue by lonix1** [gajus/eslint-plugin-jsdoc#1775]
I'm primarily using typescript, with a few javascript files here and there.

I'm using the `flat/recommended-error` preset, as I assumed that was the "main", recommended one. But I've since discovered that it has some formatting-related rules, which I don't want (I use Prettier).

There are so many presets, it's hard to understand what to use... it's actually somewhat overwhelming.

I want everything, except formatting. Which preset or presets should I use?

Thank you!

*(Also, that would make an excellent addition to the readme.)*

**Comment by brettz9** [comment]
At present, there is no way to do that for the non-TypeScript configs.

However, as one can use TypeScript flavor JSDoc on regular JavaScript files, it is generally recommended to use this so as to be able to run type-checking on JavaScript files. The preferred way for you to do this is as follows: 

```js
import {jsdoc} from 'eslint-plugin-jsdoc';

export default [
  jsdoc({config: 'flat/contents-typescript-flavor-error'}),
  jsdoc({config: 'flat/logical-typescript-flavor-error'}),
  jsdoc({config: 'flat/requirements-typescript-flavor-error'}),
];
```

The only real difference with non-TypeScript configs is that the regular TypeScript-flavored config disables `no-undefined-types` (as it is assumed you will use TypeScript do check for such types), but if you are adding the logical rules, that will include `no-undefined-types` anyways.
